batch/create-batch-tables.sql

```diff
@@ -41,7 +41,7 @@ CREATE TABLE IF NOT EXISTS `jobs` (
   PRIMARY KEY (`batch_id`, `job_id`),
   FOREIGN KEY (`batch_id`) REFERENCES batch(id) ON DELETE CASCADE
 ) ENGINE = InnoDB;
-CREATE INDEX jobs_state ON `jobs` (state);
+CREATE INDEX `jobs_state` ON `jobs` (`state`);
```

Comment:
Do you need an IF EXISTS?

Reply:
No, create_batch_tables only runs the SQL if the tables don't already exist (using jobs as a proxy): https://github.com/hail-is/hail/blob/master/batch/create-batch-tables-pod.yaml#L15